.github/workflows/doc.yml

```diff
@@ -0,0 +1,46 @@
+name: doc
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'docs/**'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Checkout Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+
+      - name: Install Requirements
+        run: |
+            sudo apt-get update && sudo apt-get install -y transifex-client
+            sudo pip install mkdocs-material mkdocs-static-i18n==0.31
+
+      - name: Push translations
+        if: ${{ github.ref == 'refs/heads/master' }}
```

Comment:
Are we sure that doesn't run for pull requests?

Reply:
I think so. No?